Sourav.txt

```diff
@@ -0,0 +1,3 @@
+Today is Tuesday.
+Date is 18 July 2022.
```

Comment:
kaun bola?

Reply:
I have updated the PR review comments please review again